integration_tests/configs/default.jsonnet

```diff
@@ -42,6 +42,7 @@
           enable: true,
           'zero-copy': true,
           'snapshot-interval': 5,
+          'cache-size': 0,
```

Comment:
`cache-size` is automatically `0` if block-stm is used.

Reply:
default is 1000 if not specify, might need change for testground as well

Reply:
https://github.com/crypto-org-chain/cronos/pull/1547

Reply:
right, seems no other place need overwrite this config